### DISCUSSION TITLE
How to override `splitright=false` for nvim-tree?

### DISCUSSION THREAD

**Discussion by dzintars** [nvim-tree/nvim-tree.lua#1332] [Q&A]
when `vim.o.splitright = false` Nvim-tree will open on the right side, which is expected global behavior.
But, i would like to keep `splitright=false` and to open nvim-tree on the left side no matter what.
How could i achieve that?
Basically i would like to replicate `NERDTreeWinPos = "left"`.

**Comment by alex-courtis** [comment]
That functionality does not exist. It could be added. PRs are always welcome ;)

**Reply by kyazdani42** [reply]
The issue is that this option is global though. Does nerd tree override it whenever it opens a new window and set it back to its previous use after ?
